static/app/components/events/searchBar.tsx

```diff
@@ -147,12 +147,12 @@ function SearchBar(props: SearchBarProps) {
         query,
         projectIdStrings,
         endpointParams,
-
-        // allows searching for tags on transactions as well
-        true,
-
-        // allows searching for tags on sessions as well
-        includeSessionTagsValues
+        {
+          // allows searching for tags on transactions as well
+          includeTransactions: true,
+          // allows searching for tags on sessions as well
+          includeSessions: includeSessionTagsValues,
+        }
```

Comment:
obj as an arg is better especially when we have so many positional args but it'd be *even* better if we replaced all of positional args instead of tacking a new one on at the end (although it makes sense why you did this).

Reply:
i can take that on. It's a lot of args but not a lot of callsites.